test_scripts/Polices/build_options/ATF_P_Register_NewApp_not_exist_inLocalPT_FinishPTU_PROPRIETARY.lua

```diff
@@ -92,6 +98,14 @@ function Test:TestStep_RegisterNewApplication()
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = "Media Application" }})
   self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
   self.mobileSession2:ExpectNotification("OnPermissionsChange")
+  self.mobileSession2:ExpectNotification("OnSystemRequest")--, {requestType = "LOCK_SCREEN_ICON_URL"} )
+  :ValidIf(function(_,data)
+      if(data.payload.requestType ~= "LOCK_SCREEN_ICON_URL") then
+        commonFunctions:printError("requestType should be PROPRIETARY")
```

Comment:
@dboltovskyi, yes you are right, but:
1. As this is general requirement as the other for OnPermissionsChange, OnAppRegistered, response of RAI request
2. In case this notification is not check in this step, there is possibility OnSystemRequest to be received in next step, that will fail the next step because of verification for one notification.